apps/dashboard/app/(app)/apis/[apiId]/keys/[keyAuthId]/[keyId]/role-toggle.tsx

```
@@ -17,53 +17,71 @@ export const RoleToggle: React.FC<Props> = ({ roleId, keyId, checked }) => {
+          <CircleCheck className="w-4 h-4 text-gray-300" />
+
+          <div className="max-w-[250px]">
+            <h2 className="font-semibold">Role added</h2>
+            <p>Changes may take up to 60 seconds to take effect.</p>
+          </div>
+
+          <div>
+            <button
+              onClick={() => disconnect.mutate({ roleId, keyId })}
+              className="bg-[#2c2c2c] text-white px-[4px] py-[2px] rounded hover:bg-[#3c3c3c] transition-colors"
+            >
```

Biome

[error] Provide an explicit type prop for the button element.

The default type of a button is submit, which causes the submission of a form when placed inside a `form` element. This is likely not the behaviour that you want inside a React application.
Allowed button types are: submit, button or reset

(lint/a11y/useButtonType)